### ISSUE TITLE
knowl text-align

### ISSUE THREAD

**Issue by alexjbest** [LMFDB/lmfdb#2732]
Knowls inherit some attributes from their container, which can cause oddities.

Specifically on an elliptic curve like <http://beta.lmfdb.xyz/EllipticCurve/Q/496/a/1> clicking any of the Local Data table headers brings up a knowl which has centered text, which looks weird IMO.

I guess this a reopening of #260 , where @haraldschilly says we can't do this in general but: "If it is something "stupidly simple", then overruling this in the general knowl definitions might be best.", I think this falls under stupidly simple!

So I propose the .knowl class should have the css attribute "text-align: left;" always.

**Comment by davidfarmer** [comment]
I am happy to work on this and other CSS issues next week.

**Comment by edgarcosta** [comment]
In your next PR we will have added ``` text-align: left;``` in 
```
.knowl-content {
  padding: 0 10px;
  text-align: left;
}
```

**Comment by alexjbest** [comment]
@edgarcosta probably .knowl is better then .knowl-content! so you get the title and the permalink too.

**Comment by edgarcosta** [comment]
Agree, I went for ```.knowl-output``` instead
```
.knowl-output {
  background: {{ color.knowl_background }};
  border: 10px solid {{ color.knowl_border }};
  {{ BORDER_RADIUS(10) }}
  padding: 0px;
  display: none;
  margin-top: 10px;
  margin-bottom: 15px;
  margin-right: 10px;
  text-align: left;
}
```

**Comment by edgarcosta** [comment]
@alexjbest, can we close this?

**Comment by alexjbest** [comment]
Fixed in #2736
